src/components/Person/Person.jsx

```diff
@@ -1 +1,14 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
```

Comment:
I'd recommend use object destructing 
```suggestion
export const Person = ({ person }) => {
  const { name, age, isMarried, partnerName , sex} = person
  
  return (<section className="Person">
```